mappings/servers/hvbc/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "hvbc",
+    "name": "hvbc",
+    "description": "Practice Server For Legit player",
+    "addresses": [
+        "hvbc.club",
+        "na.hvbc.club",
```

Comment:
As stated in the readme, there is no point in specifying subdomains here as CB already matches them automatically